js/components/wysiwyg.js

```diff
@@ -59,69 +56,6 @@ Fliplet.FormBuilder.field('wysiwyg', {
         }
       }
     },
-    placeholderLabel: function() {
```

Comment:
Why are we removing the placeholder feature?

Reply:
The placeholder feature functions properly even without the inclusion of this code.

Reply:
@tonytlwu fyi unresolved comment. Please resolve if happy with @armine-fliplet response